launcher/ui/dialogs/IconPickerDialog.cpp

```diff
@@ -64,6 +64,7 @@ IconPickerDialog::IconPickerDialog(QWidget *parent)
     // NOTE: ResetRole forces the button to be on the left, while the OK/Cancel ones are on the right. We win.
     auto buttonAdd = ui->buttonBox->addButton(tr("Add Icon"), QDialogButtonBox::ResetRole);
     auto buttonRemove = ui->buttonBox->addButton(tr("Remove Icon"), QDialogButtonBox::ResetRole);
+    buttonRemove->setObjectName("buttonRemove");
```

Comment:
just make a member in the class referencing this button to use later, so that we don't have to use the name and search it later every time :)